x/evm/keeper/grpc_query.go

```diff
@@ -336,15 +337,15 @@ func queryAddressByKeyID(ctx sdk.Context, s types.Signer, chain nexustypes.Chain
 			return types.KeyAddressResponse{}, sdkerrors.Wrap(types.ErrEVM, err.Error())
 		}
 
-		addressStrs := make([]string, len(multisigPubKey))
-		for i, address := range types.KeysToAddresses(multisigPubKey...) {
-			addressStrs[i] = address.Hex()
-		}
+		addresses := types.Addresses(slices.Map(multisigPubKey, func(p ecdsa.PublicKey) types.Address {
+			return types.Address(crypto.PubkeyToAddress(p))
+		}))
+		sort.Stable(addresses)
```

Comment:
```suggestion
		sort.SliceStable(addresses)
```
Then you don't need to implement Len/Swap/Less

Reply:
ah cool